### PR TITLE
Finish button

### PR DESCRIPTION
**Pull request by slavik-lvovsky**

### CODE REVIEW

File: frontend/src/App.vue [hunk 1]
```diff
@@ -67,7 +67,7 @@
               <v-icon left>mdi-chevron-left</v-icon>Back
             </v-btn>
             <v-btn id="next" :disabled="!stepValidated" @click="next">
-              Next
+              {{rightButtonName}}
```
Comment (tomer-epstein) [inline frontend/src/App.vue:70]: {{nextButtonText}}

File: frontend/src/App.vue [hunk 2]
```diff
@@ -146,6 +146,13 @@ export default {
     return initialState();
   },
   computed: {
+    rightButtonName() {
```
Comment (tomer-epstein) [inline frontend/src/App.vue:149]: nextButtonText() {

File: frontend/src/App.vue [hunk 3]
```diff
@@ -298,7 +305,7 @@ export default {
                 return await that.rpc.invoke("evaluateMethod", [
                   args,
                   question.name,
-                  prop
+                  prop,
```
Comment (tomer-epstein) [inline frontend/src/App.vue:308]: remove "," at end

File: frontend/src/App.vue [hunk 4]
```diff
@@ -357,7 +364,7 @@ export default {
         description: promptDescription,
         answers: {},
         active: true,
-        status: _.get(this.currentPrompt, "status")
+        status: _.get(this.currentPrompt, "status"),
```
Comment (tomer-epstein) [inline frontend/src/App.vue:367]: remove "," at end

File: frontend/src/App.vue [hunk 5]
```diff
@@ -416,13 +423,13 @@ export default {
         "generatorInstall",
         "generatorDone",
         "log",
-        "updateGeneratorsPrompt"
+        "updateGeneratorsPrompt",
```
Comment (tomer-epstein) [inline frontend/src/App.vue:426]: remove "," at end

File: frontend/src/App.vue [hunk 6]
```diff
@@ -416,13 +423,13 @@ export default {
...
       ];
       _.forEach(functions, (funcName) => {
         this.rpc.registerMethod({
           func: this[funcName],
           thisArg: this,
-          name: funcName
+          name: funcName,
```
Comment (tomer-epstein) [inline frontend/src/App.vue:432]: remove "," at end

File: frontend/src/App.vue (hunk 3 above)
Comment (slavik-lvovsky) [reply frontend/src/App.vue]: Done

File: frontend/src/App.vue (hunk 4 above)
Comment (slavik-lvovsky) [reply frontend/src/App.vue]: Done

File: frontend/src/App.vue (hunk 5 above)
Comment (slavik-lvovsky) [reply frontend/src/App.vue]: Done

File: frontend/src/App.vue (hunk 6 above)
Comment (slavik-lvovsky) [reply frontend/src/App.vue]: Done

File: frontend/src/App.vue (hunk 2 above)
Comment (slavik-lvovsky) [reply frontend/src/App.vue]: Done

File: frontend/src/App.vue (hunk 1 above)
Comment (slavik-lvovsky) [reply frontend/src/App.vue]: Done

**Pull request closed without merging**